src/audio/copier/copier.c

```diff
@@ -488,6 +488,8 @@ static int create_ipcgtw(struct comp_dev *parent_dev, struct copier_data *cd,
 	int ret;
```

Comment:
The commit message could be improved. This adds a flag, sets it in one place, but it's not used anywhere. 
Looking at review comments, the reason seems to be to separate "IPC gateway copier" and "host copier" cases. That makes more sense -> can you explain this in the git commit message? E.g. that this is a subtype of "SOF_COMP_HOST".

Reply:
ok

Reply:
ok